src/main/java/com/google/sps/servlets/AuthenticateServlet.java

```diff
@@ -79,6 +81,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse  response) thr
     Gson gson = new Gson();
 
     GivrUser user = GivrUser.getCurrentLoggedInUser();
+    // This servlet will always be called for logged in and non logged in users.
+    if (user.isModeratorOfAnyOrg()) {
```

Comment:
I removed user.setModeratingOrgs() that was originally on line 84 because getModeratingOrgs() calls setModeratingorgs().